data-types/fixed-length-ucs4/README.md

```diff
@@ -0,0 +1,33 @@
+# Fixed-length Unicode string data type
+
+Defines a data type for fixed-length Unicode strings.
```

Comment:
Probably should also include a recommendation NOT to use this type except for compatibility with existing datasets, because UTF-32 encoding is basically never a good choice.

Reply:
Probably should include a note that it is intended to only contain valid unicode strings, and that implementations may return an error if given invalid data (e.g. utf-16 surrogate pairs, out-of-range characters) if that is what is intended.

Reply:
Should include a note that NUL characters at the end are to be ignored, if that is what is intended.